lib/bmi160/BMI160.h

```diff
@@ -464,10 +544,71 @@ typedef enum {
     BMI160_ZERO_MOTION_DURATION_430_08S,        /**< 430.08 seconds */
 } BMI160ZeroMotionDuration;
 
+#define HMC_DEVADDR 0x1E
+#define HMC_RA_CFGA 0x00
+#define HMC_RA_CFGB 0x01
+#define HMC_RA_MODE 0x02
+#define HMC_RA_DATA 0x03
+
+#define HMC_CFGA_DATA_RATE_0_75 0b000 << 2
+#define HMC_CFGA_DATA_RATE_1_5  0b001 << 2
+#define HMC_CFGA_DATA_RATE_3    0b010 << 2
+#define HMC_CFGA_DATA_RATE_7_5  0b011 << 2
+#define HMC_CFGA_DATA_RATE_15   0b100 << 2
+#define HMC_CFGA_DATA_RATE_30   0b101 << 2
+#define HMC_CFGA_DATA_RATE_75   0b110 << 2
+#define HMC_CFGA_AVG_SAMPLES_1  0b00 << 5
+#define HMC_CFGA_AVG_SAMPLES_2  0b01 << 5
+#define HMC_CFGA_AVG_SAMPLES_4  0b10 << 5
+#define HMC_CFGA_AVG_SAMPLES_8  0b11 << 5
+#define HMC_CFGA_BIAS_NORMAL    0b00
+#define HMC_CFGA_BIAS_POS       0b01
+#define HMC_CFGA_BIAS_NEG       0b10
+
+#define HMC_CFGB_GAIN_0_88 0
+#define HMC_CFGB_GAIN_1_30 1 << 5
+#define HMC_CFGB_GAIN_1_90 2 << 5
+#define HMC_CFGB_GAIN_2_50 3 << 5
+#define HMC_CFGB_GAIN_4_00 4 << 5
+#define HMC_CFGB_GAIN_4_70 5 << 5
+#define HMC_CFGB_GAIN_5_60 6 << 5
+#define HMC_CFGB_GAIN_8_10 7 << 5
+
+#define HMC_MODE_HIGHSPEED 1 << 7
+#define HMC_MODE_READ_CONTINUOUS 0b00
+#define HMC_MODE_READ_SINGLEMEAS 0b01
+
+
+#define QMC_DEVADDR    0x0D
+#define QMC_RA_DATA    0x00
+#define QMC_RA_CONTROL 0x09
+#define QMC_RA_RESET   0x0B
+
+#define QMC_CFG_MODE_STANDBY    0b00
+#define QMC_CFG_MODE_CONTINUOUS 0b01
+#define QMC_CFG_ODR_10HZ        0b00 << 2
+#define QMC_CFG_ODR_50HZ        0b01 << 2
+#define QMC_CFG_ODR_100HZ       0b10 << 2
+#define QMC_CFG_ODR_200HZ       0b11 << 2
+#define QMC_CFG_RNG_2G          0b00 << 4
+#define QMC_CFG_RNG_8G          0b01 << 4
+#define QMC_CFG_OSR_512         0b00 << 6
+#define QMC_CFG_OSR_256         0b01 << 6
+#define QMC_CFG_OSR_128         0b10 << 6
+#define QMC_CFG_OSR_64          0b11 << 6
```

Comment:
QMC and HMC is not used in BMI160 directly.
Move it to 2 new files, 1 for QMC and 1 for HMC.